index.bs

```diff
@@ -711,31 +711,30 @@ QR-code and a numeric PSK), they should be for the same PSK.  If they were
 different, the PSK presenter would not know which one the user chose to use, and
 that may lead to authentication failures.
 
-Appendix [[#appendix-c]] describes two encoding schemes for PSKs that agents may support to produce either a string or a [=QR code=] for display to the user.
+Appendix [[#appendix-c]] describes two encoding schemes for PSKs that agents may
+support to produce either a string or a [=QR code=] for display to the user.
 
 Authentication with SPAKE2 {#authentication-with-spake2}
 --------------------------
 
-Issue(235): Update SPAKE2 section according to current IETF draft
-
 Issue(242): [Meta] Track CFRG PAKE competition outcome
 
 For all messages and objects defined in this section, see Appendix A for
 the full CDDL definitions.
 
 The default authentication method is
-\[SPAKE2](https://tools.ietf.org/html/draft-irtf-cfrg-spake2-08) with
+\[SPAKE2](https://tools.ietf.org/html/draft-irtf-cfrg-spake2-26) with
 the following cipher suite:
 
-1. Elliptic curve is [Curve25519](https://tools.ietf.org/html/rfc7748#page-4).
-2. Hash function is \[SHA-512](https://tools.ietf.org/html/rfc6234).
+1. Elliptic curve is \[edwards25519](https://tools.ietf.org/html/rfc7748#page-4).
+2. Hash function is \[SHA-256](https://tools.ietf.org/html/rfc6234).
 3. Key derivation function is \[HKDF](https://tools.ietf.org/html/rfc5869).
 4. Message authentication code is \[HMAC](https://tools.ietf.org/html/rfc2104).
-5. Password hash function is SHA-512.
+5. Password hash function is \[SHA-512](https://tools.ietf.org/html/rfc6234).
 
-Open Screen Protocol does not use a memory-hard hash function to hash PSKs
-with SPAKE2 and uses SHA-512 instead as the PSK is one-time use and
-is not stored in any form.
+Open Screen Protocol does not use a memory-hard hash function to hash PSKs with
+SPAKE2 and uses SHA-512 instead, as the PSK is one-time use and is not stored in
```

Comment:
Nit: you updated the hash function to SHA-256 so this comment is now out of date.

Reply:
I don't think I did, the hash function for SPAKE2 is set to `SHA-512` in point #5 above.